goldie.go

```diff
@@ -49,7 +50,7 @@ var (
 // `name` refers to the name of the test and it should typically be unique
 // within the package. Also it should be a valid file name (so keeping to
 // `a-z0-9\-\_` is a good idea).
-func Assert(t *testing.T, name string, actualData []byte) {
+func Assert(t *testing.T, name string, data interface{}, actualData []byte) {
```

Comment:
This is a breaking change that we need to find a way around. A suggestion would be to skip the `data interface{}` parameter here and pass `nil` to `compare` always. What do you think?

Reply:
I fixed it, Assert will stay as it is, another func AssertWithTemplate will accept data